### PR TITLE
Try to read IP address from local description.

### PR DESCRIPTION
**Pull request by upsuper**
Solved the issue by @ZimTester in #10

### CODE REVIEW

**Comment by ZimTester** [comment]
Thx, now it's really hard to block this. I know JS a little so my attempts were unsuccessful. And frame sandbox is working fine against addons too.
For now the only simple way I'm using to entirely block WebRTC in Chrome (without recompile) is a binary patching **chrome_child.dll** by replacing some occurrences (more simple: all occurrences) of string **`RTCPeerConnection`** with random string of the same length.

**Comment by diafygi** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -83,6 +85,18 @@ function getIPs(callback){
...
 
     }, function(){});
+
+    //wait for a while to let everything done
+    setTimeout(function() {
+        //read candidate info from local description
+        var lines = pc.localDescription.sdp.split('\n');
+
+        lines.forEach(function(line) {
+            if (line.startsWith('a=candidate:')) {
```
Comment (upsuper) [inline README.md]: It seems I forgot to make this code match the style...

Comment (upsuper) [inline README.md]: Also, it seems the "startsWith" method is added from Chrome 41. But anyway, this code doesn't need to be effective for Chrome 40 :)

Comment (diafygi) [inline README.md]: switched to indexOf in commit 782cfd27bd47a95f240f3ee2995d4959106f9146

**Comment by emanueLczirai** [comment]
@ZimTester thanks for that idea! Fastest/simplest solution! Tested to work(in linux).
eg. `sudo sed --binary -re 's|RTCPeerConnection|LoLaJvyuoN3HnJogV|g' -i /usr/lib/chromium/chromium`
I made sure that worked right by noting that the filesize didn't change; used `vbindiff` to observe the changes.
I actually made a script for myself, why not: https://github.com/emanueLczirai/coostomhuston/blob/ed5200a99bbd436fa561cfe6801327a991a8ee15/system/lenovo%20z575/OS/manjaro/filesystem%20now/home/emacs/bin/modify_chromium_exe
I don't recommend anyone use it, that's just what I use; mentioned it for consistency reasons.

**Pull request merged**